extensions/extensions-docker/aissemble-spark/src/main/resources/docker/Dockerfile

```diff
@@ -14,9 +14,7 @@ RUN mkdir $SPARK_HOME/checkpoint && \
   mkdir $SPARK_HOME/krausening && \
   mkdir $SPARK_HOME/warehouse && \
   mkdir $SPARK_HOME/policies && \
-  mkdir $SPARK_HOME/logs && \
-  useradd --home /opt/spark --group 0 --shell /usr/sbin/nologin --uid 185 spark && \
```

Comment:
Q: Is the UID the same? It's not best practice, but it's not uncommon for uid's to sneak into downstream dockerfiles.

Reply:
Yes it is the same id